idx/cassandra/cassandra_test.go

```diff
@@ -411,9 +411,7 @@ func BenchmarkIndexingWithUpdates(b *testing.B) {
 	}
 	ix.Init()
 	insertDefs(ix, b.N)
-
-	b.ReportAllocs()
-	b.ResetTimer()
+	updates := make([]*schema.MetricData, b.N)
```

Comment:
by pre-allocating you could exhaust ram. i've ran into this a few times hence the suggestion to use StopTimer and StartTimer. but anyway i guess we'll see if it happens